md/database-configuration.md

```diff
@@ -42,10 +42,6 @@ In order to configure Bonita BPM to work with your RDBMS, you need to perform th
 5. Remove h2
 6. Configure RDBMS-specific settings
 
-::: warning
-There are known issues with the management of XA transactions by MySQL engine and driver: see MySQL bugs [17343](http://bugs.mysql.com/bug.php?id=17343) and [12161](http://bugs.mysql.com/bug.php?id=12161) for more details.
-Therefore, using MySQL database in a production environment is not recommended.
-:::
 
 ::: warning
 There is also a known issue between Bitronix (the Transaction Manager shipped by Bonitasoft for the Tomcat bundle & inside Deploy bundle for Tomcat) and Microsoft SQL Server driver (refer to: [MSDN note](https://msdn.microsoft.com/en-us/library/aa342335.aspx), [Bitronix note](http://bitronix-transaction-manager.10986.n7.nabble.com/Failed-to-recover-SQL-Server-Restart-td148.html)).
```

Comment:
we could change 'There is also a known issue between Bitronix' into 'There is a known issue between Bitronix '